nemo/collections/nlp/modules/common/megatron/megatron_bert.py

```diff
@@ -41,19 +42,16 @@ class MegatronBertEncoder(BertModule):
         tokenizer_type (str): tokenizer type, currently only 'BertWordPieceLowerCase' supported.
     """
 
-    def __init__(self, model_name, config, vocab_file):
+    def __init__(self, config, vocab_file):
```

Comment:
Do we just support BERT version of Megatron or we have have plans for GPT versions too?

Reply:
only BERT for now